README.md

```diff
@@ -985,6 +985,28 @@ available
 getAttStatus: () => Promise<{status:'granted' | 'denied' | 'unknown'} | null>;
 ```
 
+### getDataConnectionInfo
+
+<kbd>App version >=14.11</kbd>
+
+Obtain metainformation about the current device data network connectivity
+
+```ts
+getDataConnectionInfo: () => Promise<{
```

Comment:
perhaps "getNetworkConnectionInfo" is a better name

Reply:
changed